packages/cli/command_spec.pony

```diff
@@ -325,22 +343,66 @@ class val ArgSpec
     _name + "[" + _typ.string() + "]" +
       if not _required then "(=" + _default.string() + ")" else "" end
 
-type _Value is (Bool | String | I64 | F64)
+class _StringSeq is ReadSeq[String]
+  let strings: Array[String]
+
+  new val empty() =>
+    strings = Array[String]
+
+  new val from(s: String) =>
+    strings = Array[String].init(s, 1)
+
+  new val combined(ss0: _StringSeq val, ss1: _StringSeq val) =>
+    let ss = ss0.strings.clone()
+    ss.append(ss1.strings)
+    strings = ss
+
+  fun string(): String =>
```

Comment:
If you're aiming to fulfill the `Stringable` interface, this should be returning a `String iso^`, which should be possible if you remove the `trn` from your `recover` statement below (so that it recovers to `iso` instead).